packages/cli/src/build.ts

```diff
@@ -16,7 +16,8 @@ type BuildOptions = {
 };
 
 export async function build({ rootDir, config, foundryProfile }: BuildOptions): Promise<void> {
-  await Promise.all([tablegen({ rootDir, config }), worldgen({ rootDir, config })]);
+  await tablegen({ rootDir, config });
+  await worldgen({ rootDir, config });
```

Comment:
this has to be done serially now that we're parsing all `*.sol` source files in worldgen to determine if a source file is a system, otherwise we get a bunch of "file does not exist" errors from tablegen deleting/creating files